File: web-exam-system-vue/src/api/WrongAnswers.js
import request from "@/util/request";

export function addWrongAnswers(data) {
  return request.post("/wrongAnswers/addWrongAnswers", data);
}
export function getWrongAnswersByUserIdAndExamId(userId, examId) {
  return request.get("/wrongAnswers/getWrongAnswersByUserIdAndExamId", {
    params: {
      userId,
      examId,
    },
  });
}

File: exam-system/src/main/java/cn/com/xuroot/entity/WrongAnswers.java
package cn.com.xuroot.entity;

import lombok.Data;

import java.util.Date;

@Data
public class WrongAnswers {
    private Integer wrongId;
    private Integer studentId;
    private Integer questionId;
    private Integer examId;
    private String wrongAnswer;
    private String correctAnswer;
    private Date collectedAt;
}
